HW1/P5/P5.txt

```diff
@@ -1 +1,38 @@
-# Your discussion here
+The shortest paths for Kevin Bacon to Harvard University and Harvard University to Kevin Bacon are presented next. The format of this output has the following logic:
+
+If we are searching the shortest distance from node 10 to node 4, the output gives all the possible shortest paths with this structure:
+
+[[4], [(4, 9), (4, 7)], [(7, 10), (9, 10)]]
+
+So, in this case there are two possible paths:
+10->9->4
+10->7->4
+
+For visual simplicity,  I split the nodes based on the distance to the given starting node in the Wikipedia data.
+
+The distance from Kevin Bacon to Harvard University is 2 and the possible paths are:
+
+[u'Harvard_University', [u'Harvard_University', u'Time_(magazine)'], [u'Harvard_University', u'Marisa_Silver'], [u'Harvard_University', u'Six_degrees_of_separation'], [u'Harvard_University', u'John_Lithgow'], [u'Harvard_University', u'College_Bowl'], 
+
+[u'College_Bowl', u'Kevin_Bacon'], [u'John_Lithgow', u'Kevin_Bacon'], [u'Marisa_Silver', u'Kevin_Bacon'], [u'Six_degrees_of_separation', u'Kevin_Bacon'], [u'Time_(magazine)', u'Kevin_Bacon']]
+
```

Comment:
Very good that you listed different paths. The path mine took is included in your list. :)